dotcom-rendering/src/components/Branding.importable.tsx

```diff
@@ -69,7 +70,7 @@ const brandingLogoStyle = css`
  * for liveblog smaller breakpoints article meta is located in the same
  * container as standfirst and needs the same styling as standfirst
  **/
-const brandingAboutLink = (palette: Palette, format: ArticleFormat) => {
+const brandingAboutLink = (format: ArticleFormat) => {
```

Comment:
This function is very complex only to add standfirst colour from mobile to desktop.

The usual pattern we would use for this would be more akin to:

```tsx

const brandingAboutLink = css`
  ${textSans.xxsmall()}
  /* other “invariant” styles, but I’d call them “common” or “base” */
  a { color: inherit; }
`

const liveBlogStyles = css`
  ${until.desktop} {
    color: ${palette('--standfirst-text')};
  }
`

const Component = ({ format }) => (
  <div
    css={[
      brandingAboutLink,
      format.design === ArticleDesign.Liveblog && liveBlogStyles;
    ]}
  >Branding</div>
)

```